src/HotelPosSystem.Tests/BookingTests.cs

```diff
@@ -0,0 +1,107 @@
+﻿using FlaUI.Core.AutomationElements;
+using FlaUI.Core.Conditions;
+using FlaUI.UIA3;
+
+namespace HotelPosSystem.Tests {
+    public class BookingTests : IDisposable, IClassFixture<ProgramFixture> {
+        private readonly UIA3Automation _automation;
+        private readonly ProgramFixture _fixture;
+
+        private const string BookingListAutomationId = "bookingList";
+
+        public BookingTests(ProgramFixture fixture) {
+            _automation = new UIA3Automation();
+            _fixture = fixture;
+        }
+
+        [Fact]
+        public void CustomerNameShouldBePresentInList() {
+            AutomationElement bookingList = GetElement(BookingListAutomationId, GetMainWindow());
```

Comment:
```suggestion
            AutomationElement bookingList = GetElement(BookingListAutomationId, GetMainWindow());
```

Denna rad upprepas i varje test. Kanske spara värdet i ett "bookingList" field eller gör en funktion för att göra alla dessa rader kortare